src/main.rs

```diff
@@ -4,15 +4,37 @@ use anyhow::{self, Context};
 use clap::Parser;
 use tokio::{select, signal};
 use tracing_subscriber::prelude::*;
-use wasmcloud_host::WasmbusHostConfig;
+
+mod config;
+
+#[allow(clippy::wildcard_imports)]
+use config::*;
+use wasmcloud_host::wasmbus::HostConfig;
 
 #[derive(Debug, Parser)]
 #[command(version, about, long_about = None)]
-struct Args;
+
+struct Args {
+    /// NATS server host to connect to
+    #[clap(long = "nats-host", default_value = DEFAULT_NATS_HOST, env = ENV_NATS_HOST)]
+    pub nats_host: String,
+    /// NATS server port to connect to
+    #[clap(long = "nats-port", default_value = DEFAULT_NATS_PORT, env = ENV_NATS_PORT)]
```

Comment:
What do we gain from indirection in all of the `DEFAULT_*` and `ENV*`? Why not define everything directly in the attribute?

Reply:
I followed the pattern we used in wash, and personally, I think it's easier to view the defaults and env vars as their own lists.